lib/filter.js

```diff
@@ -21,7 +21,11 @@ function filterPullRequest(pull_request, log) {
         log(`Ignoring #${pull_request.number}: pull request is closed`);
         return false;
     }
-    if (pull_request.draft) {
+    // Note: the `draft` boolean is a preview API which we don't enable:
```

Comment:
The reasoning for keeping this even though it doesn't work now is that it'll start working by default soon enough. If `mergeable_state` goes away after that this won't break. It might still break before then of course, but quickfix :)